bentoctl/cli/interactive.py

```diff
@@ -242,6 +242,9 @@ def select_operator():
 
 
 def select_template_type(available_templates):
+    if len(available_templates) == 1:
```

Comment:
I moved this out of this function, because it is part of business logic to select the default one, not the responsibilities of this function.

Reply:
I will revert it back and then merge